Include/pyerrors.h

```diff
@@ -219,8 +219,6 @@ PyAPI_DATA(PyObject *) PyExc_IOError;
 PyAPI_DATA(PyObject *) PyExc_WindowsError;
 #endif
 
-PyAPI_DATA(PyObject *) PyExc_RecursionErrorInst;
```

Comment:
Then it should at least be documented in What's New as having been removed since while it is an internal implementation detail, it's name wasn't given a leading `_` to more significantly signal that it wasn't meant for use outside of Python itself.